### ISSUE TITLE
Read pedigree data from VCF

### ISSUE THREAD

**Issue by timothymillar** [pystatgen/sgkit#1013]
[VCF 4.3](https://samtools.github.io/hts-specs/VCFv4.3.pdf) allows pedigree data to be stored in a VCF header. This could be translated to the `parent_id` variable.

I haven't actually come across this "in the wild" but it's something I'd like to look into.

**Comment by tomwhite** [comment]
Good idea - I haven't really looked at the pedigree stuff in VCF before.
